src/main/java/seedu/linkedout/logic/parser/ParserUtil.java

```diff
@@ -101,24 +101,24 @@ public static Email parseEmail(String email) throws ParseException {
      *
      * @throws ParseException if the given {@code tag} is invalid.
      */
-    public static Tag parseTag(String tag) throws ParseException {
-        requireNonNull(tag);
-        String trimmedTag = tag.trim();
-        if (!Tag.isValidTagName(trimmedTag)) {
-            throw new ParseException(Tag.MESSAGE_CONSTRAINTS);
+    public static Skill parseSkill(String skill) throws ParseException {
+        requireNonNull(skill);
+        String trimmedTag = skill.trim();
```

Comment:
Change variable name to reflect our app as well.
```suggestion
        String trimmedSkill = skill.trim();
```